src/stemmer.c

```diff
@@ -6,19 +6,74 @@
 #include "dep/snowball/include/libstemmer.h"
 #include "rmalloc.h"
 
-const char *__supportedLanguages[] = {"arabic",     "danish",   "dutch",     "english", "finnish",
-                                      "french",     "german",   "hungarian", "italian", "norwegian",
-                                      "portuguese", "romanian", "russian",   "spanish", "swedish",
-                                      "tamil",      "turkish",  "chinese",   NULL};
-
-int IsSupportedLanguage(const char *language, size_t len) {
-  for (int i = 0; __supportedLanguages[i] != NULL; i++) {
-    if (!strncasecmp(language, __supportedLanguages[i],
-                     MAX(len, strlen(__supportedLanguages[i])))) {
-      return 1;
+typedef struct langPair_s
+{
+  const char *str;
+  language_t lang;
+} langPair_t;
+
+langPair_t __langPairs[] = {
```

Comment:
Please prefix the enums with `RS_`, e.g. `RS_LANG_EN` or `RS_LANG_FR`, etc.

Reply:
@mnunberg 
for English and French it is clear but do we want RS_LANG_DA or RS_LANG_DU?
I think it is better to keep RS_LANG_DANISH.

Reply:
true

Reply:
well, it'd be DK and NL ;)